espnet2/text/cleaner.py

```diff
@@ -8,7 +8,11 @@
     from vietnamese_cleaner import vietnamese_cleaners
 except ImportError:
     vietnamese_cleaners = None
-
+    
+try:
+    from .korean_cleaner import Korean_cleaner
+except ImportError:
+    Korean_cleaner = None
```

Comment:
Since we have no dependency for korean cleaner, jsut using `from ... import ...` is OK without try-except.

Reply:
`vietnamese_cleaners` requires additional dependency so we use try-except.